packages/react-components/src/molecules/PeopleCard.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import css from '@emotion/css';
+import format from 'date-fns/format';
+import { UserResponse } from '@asap-hub/model';
+
+import { Card, Link, Headline2, Avatar, Caption } from '../atoms';
+import ProfilePersonalText from './ProfilePersonalText';
+import { tabletScreen } from '../pixels';
+
+const containerStyles = css({
```

Comment:
Heh, was about to comment on this in my review since it seemed overly complex. Is there a case where even no styling (`display: block`) would not suffice?

Reply:
Oh seems the behavior changed with your recent push